pom.xml

```diff
@@ -42,6 +42,11 @@
       <artifactId>logback-classic</artifactId>
       <version>${logback-classic.version}</version>
     </dependency>
+    <dependency>
```

Comment:
why to add jackson-databind if we already have Gson?

Reply:
Jackson is generally considered to be more performant and faster than Gson, especially when dealing with large amounts of data.

Reply:
So we'll be using both of them?

Reply:
I suggest to reuse the code for ScenarioSourceListenerImpl, ProxySourcesClientImpl, namely generic interface:
public interface Provider {
    <T> List<T> provideData(String resource, Class<T> valueType);
}
For this purpose, reuse the class:
![image](https://github.com/GFLCourses5/worker/assets/140727960/022392e2-d39e-44ff-8599-ae99cb2dff90)
This will allow code to be reused for reading from files, which is the best practice.